go/vt/vtorc/logic/topology_recovery.go

```diff
@@ -692,12 +686,6 @@ func executeCheckAndRecoverFunction(analysisEntry inst.ReplicationAnalysis) (err
 		// that we just completed because we would be using stale data.
 		DiscoverInstance(analysisEntry.AnalyzedInstanceAlias, true)
 	}
-	_ = AuditTopologyRecovery(topologyRecovery, fmt.Sprintf("Waiting for %d postponed functions", topologyRecovery.PostponedFunctionsContainer.Len()))
-	topologyRecovery.Wait()
-	_ = AuditTopologyRecovery(topologyRecovery, fmt.Sprintf("Executed %d postponed functions", topologyRecovery.PostponedFunctionsContainer.Len()))
-	if topologyRecovery.PostponedFunctionsContainer.Len() > 0 {
-		_ = AuditTopologyRecovery(topologyRecovery, fmt.Sprintf("Executed postponed functions: %+v", strings.Join(topologyRecovery.PostponedFunctionsContainer.Descriptions(), ", ")))
-	}
```

Comment:
Nothing ever added anything to the postponed functions. This means it was unused, and this could be all cleaned up.